_venues/portland-ballroom.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Portland Ballroom
+city: Nottingham
+---
```

Comment:
Potentially add a note that it was remodeled into the Portland Cafeteria in [date]?